plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -95,6 +143,7 @@ def complete_version(self):
     def complete_version_serialized(self):
         return tuple(version_utils.encode(field) for field in self.complete_version)
 
+    # TODO DANGER DANGER, WHAT HAPPENS WITH MongoEngine BaseDocument???
```

Comment:
@mhrivnak I added the danger TODO there as either the base class for Document or the meta class for it adds a __cmp__ method as well. I added the danger comment to make sure I went back through to ensure our version does not cause some sort of side effect in MongoEngine.